odoo/service/server.py

```diff
@@ -424,7 +424,7 @@ def run(self, preload=None, stop=False):
         """
         self.start(stop=stop)
 
-        rc = preload_registries(preload)
```

Comment:
Not needed.

Reply:
the task specifically mentions preload as an issue (https://www.odoo.com/web#id=1913129&model=project.task&view_type=form)